docs/src/index.md

```diff
@@ -15,6 +15,7 @@ To get started, please read the [Tutorial](@ref) page.
       * New function `ensemblerun!` which replaces using `replicates` in `run!`
       * New documentation page "Performance Tips"
       * Parallelization of `paramscan`
+      * New functions `dump_to_csv` and `populate_from_csv!` to save and load agent data to and from CSV files

```

Comment:
The old news have to be removed though ;) The point of this block is to bring the users attention to something really new. It is not to keep a changelog. We have a changelog that lists all incremental changes :)